src/utils/pipelineStatus.js

```diff
@@ -0,0 +1,14 @@
+// Pipeline states as defined in
```

Comment:
I wonder if `pipelineStatusValues` (or similar) would give a better indication of what the file has, and reduce the chances of name clashes

Reply:
hmm.. `pipelineStates`?

Reply:
Nah, I think `pipelineStatusValues` looks better when imported